serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -145,6 +146,11 @@ quarkus.grpc.clients.Greeter.port=50051
 
 In the same GitHub repository as the example application, there is a link:{grpc_example_url}/serverless-workflow-greeting-server-rpc-quarkus[Maven project] which provides a simple implementation of the `Greeter` service. By default, this gRPC server runs on port `50051`. Therefore, the same port is used in the `application.properties` file.
 
+=== Default enum values
+
+grpc link:{grpc_enum_url}[specification] requires enumeration types to have a 0 default value. Default value is not included in the server response payload. Therefore it is recommended to use as default a meaningless value like UNKNOWN. 
```

Comment:
what do you mean by have a 0 default value?
Isn't better just say that "requires enum types to have no default value" ?

Reply:
The fact that the default value 0 is not really relevant so I will remove that for clarity